.github/ISSUE_TEMPLATE/1_bug_report_form.yml

```diff
@@ -17,6 +17,12 @@ body:
       description: A concise description of what issue you're experiencing. **Example:** "When I click on the 'Sign in' button, I expect to be redirected to the login page, but instead I get a 404 error."
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: URL
+      description: URL of the page where the issue is observed, if applicable.
```

Comment:
Are there cases where this may not be applicable?

Reply:
Done